### ISSUE TITLE
improvement: "Run Preview" on by default in manual mode

### ISSUE THREAD

**Issue by AndrewHeumann** [DynamoDS/Dynamo#7994]
"Run Preview" is such a useful feature - visually indicating which components will recalculate on run - it should be automatically turned on by default when a dyn is in Manual run mode.

**Comment by ksobon** [comment]
I don't agree. Sure, it's a useful feature, but it just so happens to be useful when you need it. I have never found any use for it, so I would love not have it shoved down my throat for no reason.

**Comment by AndrewHeumann** [comment]
It's an orange outline, get over it

**Comment by ksobon** [comment]
Well i don't like orange. Baby blue or no deal.

**Comment by AndrewHeumann** [comment]
I'll take it
